engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -3902,6 +3919,7 @@ private Option add(Option opt)
     final Option originalAppID = add(OptionBuilder.withArgName("application id").hasArg().withDescription("Specify original application identifier for restart.").create("originalAppId"));
     final Option exactMatch = add(new Option("exactMatch", "Only consider applications with exact app name"));
     final Option queue = add(OptionBuilder.withArgName("queue name").hasArg().withDescription("Specify the queue to launch the application").create("queue"));
+    final Option tags = add(OptionBuilder.withArgName("comma separated tags").hasArg().withDescription("Specify the tags for the application").create("tags"));
```

Comment:
Since the tag content is free form (not necessarily name, value pairs), folks may potentially include different types of information and substantial information such as data serialized using other formats like JSON. Not having the ability to include comma might pose a problem. If we handle escaping the comma, then the user has to contend with escaping both space and comma. I think specifying separately would be better to start with and we can evaluate in future if we need to allow the bunching based on how users are using this feature.